internal/controllers/topology/cluster/cluster_controller.go

```diff
@@ -223,6 +242,11 @@ func (r *Reconciler) reconcile(ctx context.Context, s *scope.Scope) (ctrl.Result
 		return ctrl.Result{}, errors.Wrap(err, "error reconciling the Cluster topology")
 	}
 
+	requeueAfter := s.HookResponseTracker.EffectiveRetryAfter()
+	if requeueAfter != 0 {
+		return ctrl.Result{RequeueAfter: requeueAfter}, nil
```

Comment:
Don't we have to return an error here for the requeueAfter to take effect?:

> ```go
>	// The result.RequeueAfter request will be lost, if it is returned
>	// along with a non-nil error. But this is intended as
>	// We need to drive to stable reconcile loops before queuing due
>	// to result.RequestAfter
> ```

src: https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/internal/controller/controller.go#L328